code/WorkInProgress/zamu_mail.dm

```diff
@@ -297,28 +431,22 @@
 
 		// packages are dna-locked so you can't just swipe everyone's mail like a jerk.
 		package.target_dna = recipient["dna"]
+		package.recipient = recipient["name"]
 		package.desc = "A package for [recipient["name"]]. It has a DNA-based lock, so only [recipient["name"]] can open it."
 
+		sender = sender ? sender : "[pick(pick_string_autokey("names/first_male.txt"), pick_string_autokey("names/first_female.txt"))] [pick_string_autokey("names/last.txt")]"
+		sender = sender_title ? "[sender_title] [sender]" : sender
+		package.setup_letter_text(sender, context)
+		package.setup_stamp("[stamp_id].png", stamp_id)
```

Comment:
These are a cute idea but are going to get spammy really fast, I'd suggest making them a rare chance rather than being included with every package.